fdbserver/ptxn/TLogServer.actor.cpp

```diff
@@ -1501,6 +1501,20 @@ ACTOR Future<Void> servicePopRequest(
 	return Void();
 }
 
+ACTOR Future<Void> serveTestInfoRequest(
```

Comment:
Since there is no `wait`, this can simply be inlined at the caller side, or a non-actor function.